docs/installation.rst

```diff
@@ -14,26 +14,38 @@ Installing ``openfe``
 When you install ``openfe`` through any of the methods described below, you
 will install both the core library and the command line interface (CLI). 
 
-Installation with ``conda`` (recommended)
+Installation with ``micromamba`` (recommended)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We recommend installing ``openfe`` with ``conda``, because it provides easy
+We recommend installing ``openfe`` with ``micromamba``, because it provides easy
```

Comment:
link to micromamba install instructions? (unless I've missed it somewhere)

Reply:
Yes I just totally forgot!

Reply:
Done!